no-cft-envelopes-processed-alert.tf

```diff
@@ -0,0 +1,24 @@
+module "no-cft-envelopes-processed-alert" {
+  source            = "git@github.com:hmcts/cnp-module-metric-alert"
+  location          = "${azurerm_application_insights.appinsights.location}"
+  app_insights_name = "${azurerm_application_insights.appinsights.name}"
+
+  enabled    = "${var.env == "prod"}"
+  alert_name = "No_cft_envelopes_processed_-_Blob_Router"
+  alert_desc = "Triggers when Blob Router did not process single CFT envelope in last hour within SLA."
```

Comment:
this says last hour but query uses 70 mins ?

Reply:
I'll change the message

Reply:
I changed the time window to 1 hour